scripts/build-maven-update-version.bash

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Installs (most) maven dependencies and processes them.
+
+set -exo pipefail
+
+OPTIONS=":Z:V:"
+
+usage()
+{
+	set +x
+	USAGE=$(cat <<- USAGE_STRING
+		Usage: $(basename "${0}") [OPTION]...
+
+		Optional Arguments:
+		  -V location	Optional maven package version file. Default: \`1.0.0.SNAPSHOT\`
+		  -Z location	Optional maven cache directory. Default: \`./.m2\`
+		USAGE_STRING
+	)
+
+	echo "${USAGE}"
+
+	set -x
+}
+
+# Detect `--help`, show usage and exit
+for var in "$@"; do
+	if [ "${var}" == '--help' ]; then
+		usage
+		exit 0
+	fi
+done
+
+while getopts "${OPTIONS}" option
+do
+	case "${option}" in
+		# Optional
+    V  ) PACKAGE_VERSION="${OPTARG}";;
```

Comment:
Indentation

Reply:
Not sure why this is happening, file looks fine on my IDE

Reply:
Tabs vs spaces, we've set tabs for Bash files, 

See if you can set your editor to show the difference between a tab and space maybe?
![image](https://user-images.githubusercontent.com/2286713/143256214-7f1f6478-e51f-4f4c-be86-81520f0d6c8d.png)

Reply:
Fixed